server/handler/base.go

```diff
@@ -268,6 +271,29 @@ func (b *Base) EvaluateFetchedConfig(ctx context.Context, prctx pull.Context, cl
 	return result, nil
 }
 
+func (b *Base) ValidateChangedHeadConfig(ctx context.Context, prctx pull.Context, client *github.Client, trigger common.Trigger) error {
+	logger := zerolog.Ctx(ctx)
+
+	files, err := prctx.ChangedFiles()
+	if err != nil {
+		return err
```

Comment:
One of the conditions this can fail in is when a pull request modifies more than 3000 files. My instinct is that validation of policy changes should be best effort and that we should not fail the overall evaluation for conditions like this if the actual policy may not be impacted by the same limit.

What do you think? I can see how it might be misleading if an invalid policy resulted in a passing check due to an error with the GitHub API or something.